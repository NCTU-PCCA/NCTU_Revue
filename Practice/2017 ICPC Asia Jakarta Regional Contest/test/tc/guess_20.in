130136593380624871 51
<><><<><><><>>>><><>><>><>>>>><<<><<<<>><<><<>>><<>
