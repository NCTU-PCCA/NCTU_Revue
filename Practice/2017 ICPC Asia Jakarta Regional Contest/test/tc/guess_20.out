130136593380624871 1 130136593380624870 2 130136593380624869 130136593380624868 3 130136593380624867 4 130136593380624866 5 130136593380624865 6 7 8 9 130136593380624864 10 130136593380624863 11 12 130136593380624862 13 14 130136593380624861 15 16 17 18 19 130136593380624860 130136593380624859 130136593380624858 20 130136593380624857 130136593380624856 130136593380624855 130136593380624854 21 22 130136593380624853 130136593380624852 23 130136593380624851 130136593380624850 24 25 26 130136593380624849 130136593380624848 27
